src/vcf/meta_entry.cpp

```diff
@@ -214,6 +214,13 @@ namespace ebi
             value["Type"] != "String") {
             throw new MetaSectionError{entry.line, "INFO metadata Type is not a Integer, Float, Flag, Character or String"};
         }
+
+        // Check for predefined tag AA
+        if (value["ID"] == "AA") {
```

Comment:
Now you are touching this method, I think it would be good to use temporal variables for `value` as const references like this: `const std::string &type = value["Type"];`. That would improve speed a bit and decrease risk of misspelling "Type" and the other ones. Maybe we could even extract those hardcoded strings?